src/protocol/iterm2.rs

```diff
@@ -82,16 +83,19 @@ fn render(rect: Rect, data: &str, area: Rect, buf: &mut Buffer, overdraw: bool)
     };
 
     buf.cell_mut(render_area).map(|cell| cell.set_symbol(data));
-    let mut skip_first = false;
+
+    for x in (render_area.left() + 1)..render_area.right() {
+        if let Some(cell) = buf.cell_mut((x, render_area.top())) {
+            cell.skip = true;
+        }
+    }
```

Comment:
This is basically manual loop unrolling so we don't have to deal with the branch in the inside of each iteration of the loop

Reply:
Sounds good, my only (tiny) nitpick would be that `cell.skip = true` and `cell.set_skip(true)` should be consistent.